docs/resources/cluster.md

```diff
@@ -10,30 +9,192 @@ description: |-
 
 
 
+Enables the provisioning and management of Redpanda clusters on AWS and GCP. A cluster must always have a network and namespace and there is not much point to creating either a network or a namespace separately from a cluster.
```

Comment:
I think we can leave:
```
and there is not much point to creating either a network or a namespace separately from a cluster.
```
Out of the docs, wdyt?

Reply:
I just feel weird not including SOMETHING about why I'm not bothering to document network/namespace separate from cluster. Maybe the smarter way to handle it would be to just write the doc for network and mention that namespace is maybe EOL soon